### PR TITLE
[Editor] Update highlight cursors

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: web/images/cursor-editorTextHighlight.svg
Comment (Snuffleupagus) [inline web/images/cursor-editorTextHighlight.svg:1]: There doesn't appear to be an "outline" around the text-cursor part of the icon, is that intentional?
(Compare with the FreeText cursor, which has that.)

Comment (caseyr28) [reply web/images/cursor-editorTextHighlight.svg:1]: Good catch, UX will update to remain consistent.

**Review by Snuffleupagus — APPROVED** [review]
r=me, thanks!

**Pull request merged**